### ISSUE TITLE
are releases published to a maven repo anywhere?

### ISSUE THREAD

**Issue by wjcarpenter** [TremoloSecurity/MyVirtualDirectory#76]
I guess the releases here are just source tarballs. I found it very easy to build locally with maven. However, now I'm trying to write a custom insert, and I'd love to be able to just list a dependency on a myvd JAR that defines the interface (maybe along with transitive runtime dependencies from your pm.xml). This is something I can obviously work around, but I wanted to make sure I wasn't overlooking something. I found some stuff on maven central, but it looks like it's a couple of years old and I'm not sure what it's condition is.

**Comment by mlbiam** [comment]
Yes - https://nexus.tremolo.io/#browse/browse:releases:net%2Fsourceforge%2Fmyvd%2Fmyvd-server%2F1.0.6%2Fmyvd-server-1.0.6.pom

Makes sense to add this to the docs, will mark as a feature request

**Comment by wjcarpenter** [comment]
Doh! I guess I could have deduced that from the pom.xml that I had already downloaded with the rest of the stuff from github. Thanks for the pointer. I agree, getting it into the docs someday is a good idea.
